engine/evd.ml

```diff
@@ -14,10 +14,8 @@ open Nameops
 open Term
 open Vars
 open Environ
-open Globnames
-open Context.Named.Declaration
 
-module RelDecl = Context.Rel.Declaration
+(* module RelDecl = Context.Rel.Declaration *)
```

Comment:
I think it doesn't look bad next to the `NamedDecl` but it could go away if someone wants it to.